monai/apps/auto3dseg/data_analyzer.py

```diff
@@ -41,7 +41,8 @@ def strenum_representer(dumper, data):
     return dumper.represent_scalar("tag:yaml.org,2002:str", data.value)
 
 
-config_parser.yaml.SafeDumper.add_multi_representer(StrEnum, strenum_representer)
+if optional_import("yaml")[1]:
+    config_parser.yaml.SafeDumper.add_multi_representer(StrEnum, strenum_representer)
```

Comment:
@wyli I think maybe we should not include this YAML feature in this release?

Thanks.

Reply:
Let me double check @Nic-Ma . Thanks for bringing it up

Reply:
it is needed for exporting analyzer's yaml output, I added `if optional_import("yaml")[1]:` otherwise it can't pass the min tests for some reason.

Reply:
The multi_representer is a necessary line to avoid the str(MetaKeys) calls across all the scripts. 

The added optional_import seems to be a fix to tell if pyyaml is available. I remember the StrEnum works okay with json so pure JSON use case is possible.